apps/condo/lang/ru/ru.json

```diff
@@ -2196,6 +2196,8 @@
   "api.billing.sendNewReceiptMessagesToResidentScopes.INVALID_BILLING_CATEGORY_PROVIDED": "Необходимо передать id существующей биллинговой категории",
   "api.billing.sendNewReceiptMessagesToResidentScopes.SCOPES_IS_EMPTY": "Необходимо передать хотя бы один элемент в scopes",
   "api.billing.sendNewReceiptMessagesToResidentScopes.INVALID_BILLING_PROPERTY_PROVIDED": "Все переданные id домов (billingProperty) в скоупах должны принадлежать организации из переданного биллингового контекста",
+  "api.billing.registerBillingReceiptFile.MULTIPLE_BILLING_RECEIPTS_FOUND": "Найдено несколько квитанций для PDF файла",
```

Comment:
I'd rather: found more then 1 billing receipt, as it is more specific

Reply:
Done